layers/meta-balena-raspberrypi/conf/machine/revpi-connect.conf

```diff
@@ -7,4 +7,5 @@ require conf/machine/include/revpi.inc
 KERNEL_DEVICETREE:append = " \
 	overlays/revpi-connect.dtbo \
 	overlays/revpi-connect-dt-blob-overlay.dtb \
+	overlays/revpi-connect-can.dtbo
```

Comment:
backslash needed here

Reply:
Thank you, I'll fix it.

Reply:
Thanks @adolfogc , the build fails currently and it looks like it's because the dts hasn't been added to the Makefile:

```
/1_4.19.95+gitAUTOINC+54d60ba30a-r0/recipe-sysroot=                      -fdebug-prefix-map=/work/build/tmp/work/revpi_connect-poky-linux-gnueabi/linux-kunbus/1_4.19.95+gitAUTOINC+54d60ba30a-r0/recipe-sysroot-native=  -fdebug-prefix-map=/work/build/tmp/work-shared/revpi-connect/kernel-source=/usr/src/kernel   LD=arm-poky-linux-gnueabi-ld.bfd   LOADADDR=0x00008000
| make[3]: *** No rule to make target 'arch/arm/boot/dts/overlays/revpi-connect-can.dtbo'.  Stop.
| arch/arm/Makefile:345: recipe for target 'overlays/revpi-connect-can.dtbo' failed
| make[2]: *** [overlays/revpi-connect-can.dtbo] Error 2
| Makefile:146: recipe for target 'sub-make' failed
| make[1]: *** [sub-make] Error 2
| Makefile:24: recipe for target '__sub-make' failed
```

Here's an example on how to add it: https://github.com/balena-os/balena-raspberrypi/blob/master/layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi/0001-Add-tpm-slb9670-tis-spi-DT-overlay.patch#L21

Reply:
Since this overlay is for revpi solely and you are modifying the linux-kubus Makefile, you'll need to append this to the revpi-connect, as it most probably won't apply the raspberrypi kernel Makefile:

```
KERNEL_DEVICETREE:append:revpi-connect = ...
SRC_URI:append:revpi-connect = ...
```

Reply:
Hi @acostach, thanks for the feedback. I'm not following the last part, do you mean I should add it like this:
```
SRC_URI:append:revpi-connect = " \
    file://0001-Add-revpi-connect-can-overlay.patch \
"
```